### PR TITLE
To-Do List: interactive list

### PR DESCRIPTION
**Pull request by migo1**
### MILESTONES

- Use webpack to bundle JavaScript.
- Learn how to use proper ES6 syntax.
- Use ES6 modules to write modular JavaScript.
- add some functionality to the application to make it interactive.

### CODE REVIEW

File: src/module/event.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  toDoTask.completed = checkbox.checked;
+  localStorage.setItem("todoList", JSON.stringify(items));
+  if (checkbox.checked) {
+    text.style.textDecoration = "line-through";
+    // icon.style.display = "block";
+    // trashIcon.style.display = "none";
+  } else {
+    text.style.textDecoration = "none";
+    // trashIcon.style.display = "block";
+    // icon.style.display = "none";
```
Comment (elisha2kyakpo1) [inline src/module/event.js:11]: - [ ] Kindly remove all the commented code in order to make the codebase clean.

File: src/module/event.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  localStorage.setItem("todoList", JSON.stringify(tasks));
+  toDo.displayList();
+};
+
+// ...existing code...
+
+// const initializeClearButton = () => {
+//   const clearButton = document.getElementById("clear");
+//   clearButton.addEventListener("click", clearCompletedTasks);
+// };
```
Comment (elisha2kyakpo1) [inline src/module/event.js:26]: - [ ] Similarly, as mentioned above, remove all the commented code in order to make the codebase clean.

File: src/module/toDoList.js [hunk 3]
```diff
@@ -51,19 +62,11 @@ export default class ToDoList {
         const taskIndex = toDoTask.index;
         this.deleteTask(taskIndex);
 
-        icon.style.display = "block";
-        trashIcon.style.display = "none";
+        // icon.style.display = "block";
+        // trashIcon.style.display = "none";
```
Comment (elisha2kyakpo1) [inline src/module/toDoList.js:66]: - [ ] We have to clean the codebase by removing all the commented code.

**Review by elisha2kyakpo1 — CHANGES_REQUESTED** [review]
Hi 👋 @migo1, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ The PR title and description are good and professional.
✔️ No linter errors.
✔️ Descriptive readme file.
✔️ The checkbox interaction is done properly👏
👍 The clear all completed button works very well.
👏 Tha app works as expected.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by migo1** [comment]
@elisha2kyakpo1 noted

**Review by LionRouge1 — APPROVED** [review]
Hi @migo1, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**